ceasiompy/SU2Run/tests/tests_su2actuatordisk/test_su2actuatordisk.py

```diff
@@ -133,6 +172,42 @@ def test_save_plots(tmp_path):
     assert Path(tmp_path, "propeller_test", "prandtl_correction.png").exists()
 
 
+def test_check_values():
+    radial_stations = get_radial_stations(1, 0.2, number_of_stations=5)
+    radial_stations_spacing = radial_stations[1] - radial_stations[0]
+
+    input_values = {
+        "test1": [
+            [
+                radial_stations_spacing,
+                np.array([0.1, 0.2, 0.3, 0.4, 0.2]),
+                np.array([0.07, 0.17, 0.25, 0.32, 0.22]),
+                100,
+                0.9,
+                radial_stations,
+                2,
+                20,
+            ],
+            [0.24, 0.20600, 40691.358, 0.3973, 0.7725],
+        ],
+    }
+
+    for values in input_values.values():
+        (
+            total_power_coefficient,
+            optimal_total_thrust_coefficient,
+            thrust_density_ratio,
+            computed_total_thrust_coefficient,
+            eta,
+        ) = check_values(*values[0])
+
+        assert total_power_coefficient == pytest.approx(values[1][0], rel=1e-3)
+        assert optimal_total_thrust_coefficient == pytest.approx(values[1][1], rel=1e-3)
+        assert thrust_density_ratio == pytest.approx(values[1][2], rel=1e-3)
+        assert computed_total_thrust_coefficient == pytest.approx(values[1][3], rel=1e-3)
+        assert eta == pytest.approx(values[1][4], rel=1e-3)
+
+
 def test_thrust_calculator():
```

Comment:
Check how the "tmp_path" is use in the function  `test_add_damping_derivatives` in "ceasiompy/SU2Run/tests/tests_su2config/test_su2config.py" I think it will simplify the test